src/engine/ruleset/decoders/fortinet/fortimail.yml

```diff
@@ -0,0 +1,200 @@
+---
+name: decoder/fortinet-fortimail/0
+
+metadata:
+  title: Decoder for Fortinet fortimail
+  description: Decoder for Fortinet fortimail
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.3
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/09
+  references:
+    - https://docs.fortinet.com
+
+sources:
+  - decoder/queue-localfile/0
+
+parse:
+  logpar:
+    # date=2018-1-27 time=05:21:06 device_id=atcupid log_id=onse log_part=psa type=virus_file-signature pri=high destla to="fugitse" src=[10.12.86.130] session_id=dese msg="Attachment file (duntutla) has sha1 hash value: lamco"
+    - event.original: >-
```

Comment:
This could not be done. I found the following log line that does not allow it:
`date=2018-11-23 time=09:15:06 device_id=imipsam log_id=eumiu log_part=tatevel type=event subtype=smtp pri=high user=quisnostui=sequines(10.115.154.104) action=cancelstatus=lorumsession_id="suntexpl" msg="DSN: to <<iqu>; reason:success; sessionid:tatis"`

Reply:
this part of the log eliminates the possibility of kv:
`action=cancelstatus=lorumsession_id="suntexpl"`